### PR TITLE
add location of birth labels to summary fields

### PR DESCRIPTION
**Pull request by cibelius**
## Description

Core PR: https://github.com/opencrvs/opencrvs-core/pull/9587
https://github.com/opencrvs/opencrvs-core/issues/9385

* Configure labels for birth summary, so it displays 'Location of birth' instead of 'Place of birth'
* Configure the empty text to always be 'No location of birth'

## Checklist

- [x] I have linked the correct Github issue under "Development"
- [x] I have tested the changes locally, and written appropriate tests
- [x] I have tested beyond the happy path (e.g. edge cases, failure paths)
- [ ] I have updated the changelog with this change (if applicable)
- [x] I have updated the GitHub issue status accordingly

### CODE REVIEW

File: src/form/v2/birth/index.ts [hunk 1]
```diff
@@ -74,7 +79,7 @@ export const birthEvent = defineConfig({
       {
         fieldId: 'child.birthLocation',
         emptyValueMessage: {
-          defaultMessage: 'No place of birth',
+          defaultMessage: 'No location of birth',
```
Comment (makelicious) [inline src/form/v2/birth/index.ts:82]: I understood that the 'No place of birth' is the correct one. See pic from V1:

Screenshot 2025-05-19 at 9 59 32

File: src/form/v2/birth/index.ts [hunk 2]
```diff
@@ -60,10 +60,15 @@ export const birthEvent = defineConfig({
       {
         fieldId: 'child.placeOfBirth',
         emptyValueMessage: {
-          defaultMessage: 'No place of birth',
+          defaultMessage: 'No location of birth',
           description: 'This is shown when there is no child information',
           id: 'v2.event.birth.summary.child.placeOfBirth.empty'
         },
+        label: {
+          defaultMessage: 'Location of birth',
```
Comment (makelicious) [inline src/form/v2/birth/index.ts:68]: I think this is the actual fix. Without explicit label it takes 'Place of delivery' as the label (what is defned in the form). 

Other way would be to use a duplicate field e.g. `child.birthLocation` but I think this is better.

**Review by makelicious — APPROVED** [review]
